chia/wallet/did_wallet/did_info.py

```diff
@@ -12,11 +12,12 @@
 @dataclass(frozen=True)
 @streamable
 class DIDInfo(Streamable):
-    origin_coin: Optional[Coin]  # puzzlehash of this coin is our DID
+    origin_coin: Optional[Coin]  # Coin ID of this coin is our DID
```

Comment:
Unless I'm misunderstanding what this is and it's not appropriate, consider using the name "launcher_id" or something else with "launcher" in it.